staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -110,16 +73,9 @@ func loadWebhook(configFile string, groupVersion schema.GroupVersion) (*webhook.
 		[]schema.GroupVersion{groupVersion}, defaultInitialBackoff)
 }
 
-func newBlockingWebhook(configFile string, groupVersion schema.GroupVersion) (*blockingBackend, error) {
-	w, err := loadWebhook(configFile, groupVersion)
-	if err != nil {
-		return nil, err
-	}
-	return &blockingBackend{w}, nil
-}
-
 type blockingBackend struct {
-	w *webhook.GenericWebhook
+	w     *webhook.GenericWebhook
+	retry bool
```

Comment:
Please remove this flag and always retry

Reply:
ok